samples/tab-mail/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-mail-ts",
+    "source": "officeDev",
+    "title": "Tab Mail In Outlook",
+    "shortDescription": "This sample helps to compose mail from perosnal tab",
```

Comment:
This sample app shows how to compose mail in Outlook using personal tab app.

Reply:
fixed